README.md

```diff
@@ -262,7 +262,7 @@ The authors decided to use elliptic curve over RSA, because:
 * Curve 25519 is widely used and [respects several criteria](https://safecurves.cr.yp.to/)
 * Common arguments, such as the ones of [Trail of bits](https://blog.trailofbits.com/2019/07/08/fuck-rsa/)
 
-AES-GCM is used because it is one of the most commonly used AEAD algorithms and using one avoids a whole class of attacks. In addition, we can rely on hardware acceleration (like AES-NI) to keep reasonable performance.
+AES-GCM is used because it is one of the most commonly used AEAD algorithms and using one avoids a whole class of attacks. In addition, it allows us to rely on hardware acceleration (like AES-NI) to keep reasonable performance.
```

Comment:
```suggestion
AES-GCM is used because it is one of the most commonly used AEAD algorithms and using one avoids a whole class of attacks. In addition, it lets us rely on hardware acceleration (like AES-NI) to keep reasonable performance.
```